packages/form/src/form/FormSection/renderField.tsx

```diff
@@ -275,7 +276,9 @@ const renderField = (props: RenderFieldProps) => {
         )
       }
       default: {
-        const title = get(item, name)
+        const title = formatDateTimeValue
+          ? formatDateTimeValue(item)
```

Comment:
rename to setTitle instead because there is no context of the concept DateTime around here